src/main/java/org/commcare/cases/instance/StorageBackedChildElement.java

```diff
@@ -228,7 +228,7 @@ public TreeElement getAttribute(QueryContext context, String namespace, String n
     }
 
     @Override
-    public TreeElement getAttribute(String namespace, String name) {
+    public AbstractTreeElement getAttribute(String namespace, String name) {
```

Comment:
Why was this change necessary? It looks like the downstream method returns a treeelement anyway.